docs/codeql/codeql-cli/creating-codeql-databases.rst

```diff
@@ -168,7 +168,9 @@ generate a database, therefore the build method must be available to the CLI.
 Detecting the build system
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The CodeQL CLI includes autobuilders for C/C++, C#, Go, and Java code. CodeQL
+.. include:: ../reusables/kotlin-beta-note.rst
+
+The CodeQL CLI includes autobuilders for C/C++, C#, Go, Java and Kotlin code. CodeQL
```

Comment:
Do we also need an example build command for Kotlin for this section?

Reply:
No probs!

> I guess we could update "Java project built using Gradle" to "Java and Kotlin project built using Gradle".

This seems like a possibility, but possibly not needed?

Reply:
> This seems like a possibility, but possibly not needed?

I agree. I think it's fine to leave it as is.